TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/NodeType.php

```diff
@@ -125,6 +125,11 @@ public function __construct($name, array $declaredSuperTypes, array $configurati
         }
 
         $this->localConfiguration = $configuration;
+
+        $this->fullConfiguration = $fullConfiguration;
```

Comment:
yeah, will recheck it again. The idea setting it to null was to save memory as the config is no longer needed when the node types are built.